content/case_histories/bookpurnong/synthesis.rst

```diff
@@ -3,3 +3,45 @@
 Synthesis
 =========
 
+At the step of synthesis, geophysicists need to connect the interpretation results to the geologic questions asked at set-up, and also determine whether iterations of the previous steps are necessary. At Bookpurnong, the hydrologists are primarily concerned about the salinity of the groundwater that indicates the recharge (gaining) or discharge (losing) status of the river. Using the inversion models shown in Interpretation, both models consistently indicate that the upstream part of the river is fresher (less conductive) and thus losing water; the downstream part of the river is saliner (more conductive) and thus gaining water. 
+
+In order to achieve a more robust interpretation, we compare the models obtained from RESOLVE and SkyTEM.
+
+.. figure:: ./images/booky-comparemodel1.jpg
+    :align: left
+    :scale: 80% 
+    :name: booky-comparemodel1
+
+On the A-B cross section, both systems show similar relative distribution of conductivity for a wavy conductive layer that is likely to be associated with the saline water. However, we notice that there are some difference:
+
+(1) The overall conductivity in RESOLVE is higher than in SkyTEM.
+
+(2) The depth of the saline water layer is shallower in RESOLVE than in SkyTEM. 
+
+The discrepancy is more clearly evidenced on the depth slice maps at elevation 10 m and 0 m. The conductivity of RESOLVE at 10 m is actually similar to that of SkyTEM at 0 m. 
+    
+.. figure:: ./images/booky-comparemodel2.jpg
+    :align: left
+    :scale: 80% 
+    :name: booky-comparemodel2    
+    
+.. figure:: ./images/booky-comparemodel3.jpg
+    :align: left
+    :scale: 80% 
+    :name: booky-comparemodel3    
+
+Possibilities are:
+
+(1) Systematic errors in the data. For example, the flight height could be incorrect.
+
+(2) Naturally occuring variation of the conductivity or due to SIS.
+
+(3) The weaker conductivity in SkyTEM may be compensated by the thickness. 
+
+So further investigation may be needed to reconcile the models.
```

Comment:
Boreholes to give local reference and constrains? it seemed we had some in the region